feature/policy_forwarding/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,234 @@
+# PF-1.6: Policy forwarding based GUE tunnel decap to default and non-default network-instance
+
+## Summary
+This test ensures NOS is able to host multiple VRFs, perform GUE tunnel DECAP in the default VRF and also allows for gradual traffic migration from Default to Non-Default VRF using VRF selection policy.
+
+
+## Test environment setup
+
+### Topology
+Create the following connections:
+```mermaid
+graph LR; 
+subgraph ATE [ATE]
+    A1[Port1]
+    A2[Port2] 
+end
+subgraph DUT1 [DUT1]
+    B1[Port1]
+    B2[Port2]
+end
+A1 <-- IBGP(ASN100) & ISIS --> B1; 
+A2 <-- EBGP(ASN100:ASN200) --> B2;
+```
+
+### Configuration generation of DUT and ATE
+
+#### DUT Configuration
+* Configure ISIS[Level2] and IBGP[ASN100] as described in topology between ATE:Port1 and DUT:Port1
+* Configure EBGP[ASN200] between ATE:Port2 and DUT: Port2
+* Configure route leaking from the default VRF and non-default VRF and vice versa.
+* Configure a policy based traffic steering from default to Non Default VRF, this policy should be able to steer the traffic frmom Default VRF to non default VRF and vice vers based on teh destination IP/IPV6 address.
+* DUT has the following VRF selection policy
```

Comment:
Why does the policy has the punt rules to the two VRFs at the same time?

I.e., if the policy is applied to an interface in DEFAULT vrf, it can only redirect something to the non-default vrf (otherwise its just the regular routing?)
If the policy is applied to an interface in the non-default vrf, it can only redirect something to the default VRF. Am I missing something?